www/src/components/thumbprint-components/component-header/platform-nav.jsx

```diff
@@ -32,21 +33,32 @@ const getPlatformDisplayName = path => {
         usage: 'Usage',
         react: 'React',
         scss: 'SCSS',
-        ios: 'iOS',
+        ios: 'iOS (Old)',
```

Comment:
Nit: Should use UIKit instead of old. Reader should infer that automatically.

Reply:
+1, I'd probably not strikethrough the UIKit one since we'll still be using it for a long time to come.

Reply:
@egoens I didn't realize that banner thing existed.  I'll switch to using that instead.